src/main/java/equipments/EquipmentDescription.java

```diff
@@ -0,0 +1,26 @@
+package equipments;
+
+import team.ITeamMember;
+
+import java.util.List;
+import java.util.Map;
+
+public class EquipmentDescription {
```

Comment:
Functional classes should be named with an adjective describing their main action- for example, this is a `describer`, not a `description`